.github/workflows/daily-deploy-production.yml

```
@@ -96,6 +97,39 @@ jobs:
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
 
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    needs: notify-start
```

```suggestion
    needs: validate-build-status
```
I think we only want the build job to run if the build status is successful. `notify-start` isn't dependent on `validate-build-status`